cadc-log/src/main/java/ca/nrc/cadc/log/WebServiceLogInfo.java

```diff
@@ -165,6 +166,29 @@ private String getPreamble() {
         return sb.toString();
     }
 
+    // handle Lists, refer to GroupLogInfo.addedMembers
+    // currently only List<String> is handled
+    private void populateList(StringBuilder sb, Object o) {
+        sb.append("[");
```

Comment:
Were you able to test that the square brackets work with the elasticsearch/kibana log scraping?

Reply:
No, but this is in the JSON spec. Elasticsearch should handle it. I suppose we can check kabana log to see if it handles an empty list. If there an easy to test this out on Elasticsearch?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/null-value.html, and https://stackoverflow.com/questions/43326775/elastic-search-query-empty-list
indicates that ElasticSearch handles an empty list []